### ISSUE TITLE
Steps item body displays none when refreshing page

### ISSUE THREAD

**Issue by xfakir** [buefy/buefy#3046]
all of the steps item bodys are added style ‘display:none;' after refreshing page
![image](https://user-images.githubusercontent.com/22212420/99393147-dfc2bc00-2917-11eb-8e08-356ce5f2d827.png)

**Comment by xfakir** [comment]
help me  plz!

**Comment by jtommy** [comment]
Can you reproduce it on codesanbox or similar?

**Comment by RyanShahidi** [comment]
I ran into this same issue and re-created it in this codesandbox: 

https://codesandbox.io/s/nuxt-buefy-customize-forked-04tsl?file=/pages/buefy-steps-error.vue

To reproduce the issue, click the link I placed at the bottom to take you to the issue from the homepage. The steps should appear normally, with the limited information presented. When you are on the issue page, click the refresh button to re-load the page. The page will be re-loaded with all of the information in the steps hidden and the same style that the above poster noted. I have not had this issue when navigating to the page with steps from within the SPA, only on hard refreshes of the page.
